app/models/base.rb

```diff
@@ -31,6 +31,14 @@ def get_service_ids(appliance_id)
       sites_and_apps.select! do |site|
         !site['appliance'].select! { |appliance| appliance['id'] == appliance_id}.blank?
       end
-      sites_and_apps.collect { |service| service['service_id'] }
+      sites_and_apps.collect { |service| service['id'] }
+    end
+
+
+    def app_and_serv_check(appliance_id, service_id)
```

Comment:
In that case...is it a best practice? Question more for @arax than for you I guess :smile: